reconcile/cli.py

```diff
@@ -229,8 +238,8 @@ def init_log_level(log_level):
 def integration(ctx, configfile, dry_run, log_level):
     ctx.ensure_object(dict)
 
-    init_log_level(log_level)
     config.init_from_toml(configfile)
+    init_log_level(log_level)
```

Comment:
i think we should start thinking about sretoolbox.logging
out of scope of this PR of course.
@apahim, wdyt?

also feel free to take a look at this PR!